### DISCUSSION TITLE
Realm migration ruleset error

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#16624] [Q&A] [answered]
currently when i open osu it throws some error (running on master branch 68ee80c5deb4accb1f069e38ba527c322e05aff4)
i dont have any rulesets in the ruleset folder atm but i had a few back then.
[database.log](https://github.com/ppy/osu/files/7939611/database.log)
[network.log](https://github.com/ppy/osu/files/7939612/network.log)
[performance.log](https://github.com/ppy/osu/files/7939613/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/7939614/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7939615/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/7939616/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/7939617/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/7939618/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7939619/updater.log)

client db: [client.zip](https://github.com/ppy/osu/files/7939658/client.zip)

**Comment by peppy** [comment, marked as answer]
Are you sure that's the correct `client.db` file? I can't seem to reproduce the failure.

**Reply by mouzedrift** [reply]
yeah the `client.db` file seems to be correct something i've noticed after double checking is that this error only occurs after opening the client settings it seems.

**Reply by peppy** [reply]
Can you give the fix at #16627 a try to confirm it works for you? Seems to with your database loaded locally.

**Reply by mouzedrift** [reply]
The fix works fine i don't get any errors anymore.
